packages/app/models/order/types.ts

```diff
@@ -1,20 +1,6 @@
-import { Token } from "@/models/token/types";
-
-export interface Order {
-  id: string;
-  owner: string;
-  receiver: string;
-  amount: string;
-  sellToken: Token;
-  buyToken: Token;
-  createdAt: string;
-  startTime: number;
-  endTime: number;
-  orderSlots: string[];
-  cancelledAt: string | null;
-  interval: string;
-}
+import { Order as StacklyOrder } from "@stackly/sdk/dist/vaults/subgraph";
 
+export interface Order extends StacklyOrder {}
```

Comment:
This is cleaner.

```suggestion
import { Order } from "@stackly/sdk/dist/vaults/subgraph";
```

Reply:
Also, we should not be importing this from `dist` folder. Should come directly from `@stackly/sdk` like ChainId.

Reply:
I don't think it's exported from `SDK`

Reply:
Exported type Order from Sdk